website/content/docs/v0.15/Reference/configuration.md

```diff
@@ -346,6 +346,40 @@ kubelet:
 ```
 
 
+</div>
+
+<hr />
+<div class="dd">
+
+<code>pods</code>  <i>[]Unstructured</i>
+
+</div>
+<div class="dt">
+
```

Comment:
One more thing in here, maybe worth mentioning the same from the release notes that you can update these without a reboot.